libs/ts-rest/nest/src/lib/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const TsRestAppRouteMetadataKey = Symbol('ts-rest-app-route');
+export const JsonQuerySymbol = Symbol('ts-rest-json-query');
+export const ValidateResponsesSymbol = Symbol('ts-rest-validate-responses');
```

Comment:
Separated symbols into separate file because we had an unnecessary circular dependency between `ts-rest.decorator.ts` and `ts-rest.interceptor.ts`... but also cleaner 😄